third_party/3/aiofiles/base.pyi

```diff
@@ -0,0 +1,39 @@
+from types import CodeType, FrameType, TracebackType, coroutine
+from typing import Any, Coroutine, Generator, Generic, Iterator, Optional, Type, TypeVar, Union
+
+_T_co = TypeVar("_T_co", covariant=True)
+_V_co = TypeVar("_V_co", covariant=True)
+_T_contra = TypeVar("_T_contra", contravariant=True)
+
+class AsyncBase:
+    def __init__(self, file: str, loop: Any, executor: Any) -> None: ...
+    async def __aiter__(self) -> Iterator[str]: ...
+    async def __anext__(self) -> str: ...
+
+class _ContextManager(Generic[_V_co, _T_co, _T_contra], Coroutine[_V_co, _T_co, _T_contra]):
+    def __init__(self, __coro: Coroutine[_V_co, _T_co, _T_contra]) -> None: ...
+    def send(self, __value: _T_contra) -> _T_co: ...
```

Comment:
Mypy is unhappy about a number of incompatible overrides here. I think you should be able to remove most methods from this class, since they're the same as in the Coroutine base class.